vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -59,7 +59,10 @@ import hudson.AbortException
      * as key and the corresponding value as value. The value can also be
      * a nested structure.
      * The properties will be added to the pod spec inside node `spec` at the
-     * same level like e.g. `containers`.
+     * same level like e.g. `containers`, `imagePullSecrets`
```

Comment:
I don't think we need to add `imagePullSecrets` here...
The properties (imagePullSecrets) will be added to the same level as `containers`. It is already clear :)